python/vyos/ifconfig/bridge.py

```diff
@@ -287,6 +287,9 @@ def update(self, config):
                     value = interface_config.get('priority')
                     lower.set_path_priority(value)
                 
+                if 'wlan' in interface:
```

Comment:
Why not leave the code as it was before? This feels like an unnecessary change which probably does more harm then good.